### PR TITLE
fix: move SigBlockHeight from TxBody into AuthInfo

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

closes: #293 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,6 +50,7 @@
 * (wasm) [\#253](https://github.com/line/lbm-sdk/pull/253) remove MaxGas const
 * (wasm) [\#254](https://github.com/line/lbm-sdk/pull/254) Specify wasm event types
 * (x) [\#255](https://github.com/line/lbm-sdk/pull/255) Remove legacy from modules
+* (proto) [\#293](https://github.com/line/lbm-sdk/pull/293) Is it appropriate for sig_block_height to be included in the tx_body?
```
Comment (egonspace) [inline CHANGELOG.md:53]: Wouldn't PR-title be better than issue-title for this description?

**Review by egonspace — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -51,6 +51,7 @@
 * (wasm) [\#254](https://github.com/line/lbm-sdk/pull/254) Specify wasm event types
 * (x) [\#255](https://github.com/line/lbm-sdk/pull/255) Remove legacy from modules
 * (perf) [\#320](https:/github.com/line/lbm-sdk/pull/320) internal objects optimization (BaseAccount, Balance & Supply)
+* (proto) [\#344](https://github.com/line/lbm-sdk/pull/344) move SigBlockHeight from TxBody into AuthInfo
```
Comment (whylee259) [inline CHANGELOG.md:54]: How about changing the scope to `auth` rathe than `proto`?

**Review by whylee259 — APPROVED** [review]
Congratulations on your first PR. 
I have left a minor comment, but it's LGTM.
And how about changing the title of this PR starting `feat` rather than `fix`?
This PR contains changes that required consumer actions.

File: CHANGELOG.md (hunk 2 above)
Comment (0Tech) [reply CHANGELOG.md:54]:
```suggestion
* (auth) [\#344](https://github.com/line/lbm-sdk/pull/344) move SigBlockHeight from TxBody into AuthInfo
```

**Pull request merged**